python/apertium_core.i

```diff
@@ -20,14 +20,14 @@ public:
   void transfer_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
 };
 
-class apertium_tag: public Apertium::apertium_tagger
+class tag: public Apertium::apertium_tagger
```

Comment:
This should still be `tagger`. Classes should be nouns, functions should be verbs.